Libraries/process/Sources/DefaultLauncher.cpp

```diff
@@ -101,6 +121,24 @@ launch(Filesystem *filesystem, Context const *context)
         return ext::nullopt;
     } else {
         /* Fork succeeded, existing process. */
+        if (pipe_setup_success) {
+            close(pfd[1]);
+            /* Read child's stdout/stderr through pipe, and output stdout */
+            while (true) {
+                char pin;
+                int readlen = read(pfd[0], &pin, 1);
```

Comment:
You'll have pretty bad performance with this. Consider the following code instead:
```
#define BUF_SIZE 1024
char pin[BUF_SIZE];
int readlen = read(pfd[0], &pin, sizeof(pin));
if (readlen > 0) {
    fwrite(pin, readlen, 1, stdout);
} else {
    // ...
```